test/integration/clone-cleanup.js

```diff
@@ -4,7 +4,7 @@ const makeTestStorage = require('../fixtures/make-test-storage');
 const readFileToBuffer = require('../fixtures/readProjectFile').readFileToBuffer;
 const VirtualMachine = require('../../src/index');
 
-const projectUri = path.resolve(__dirname, '../fixtures/clone-cleanup.sb2');
+const projectUri = path.resolve(__dirname, '../fixtures/clone-cleanup.sb3');
```

Comment:
I'm a little concerned about this change with clone cleanup becoming an sb3 project instead of an sb2. Doesn't this test exist in part to check compatibility with 2.0 behavior?